sdk/cpp/core/src/netconf_service.hpp

```diff
@@ -34,6 +34,8 @@
 #include "netconf_provider.hpp"
 #include "types.hpp"
 
+using namespace std;
```

Comment:
Suggest not to add `using namespace` statements in `.hpp` files. See above comment.